test/blackbox-tests/test-cases/no-infer/error-without-no-infer/dune

```diff
@@ -0,0 +1,6 @@
+(rule
```

Comment:
Nowadays we prefer to create such files directly from the `run.t` as follow:

```
   $ cat >dune<<EOF
   > (rule
   >  ...
   > EOF
```

This is the keep the test more self-contained. It makes it easier to follow the logic of the test by just looking at the `run.t` file.

Could you do that for this test?

Apart from that, it all looks good to me.